animation_nodes/nodes/animation/delay_time.py

```diff
@@ -20,4 +26,7 @@ def drawLabel(self):
         else: return "Delay Time"
 
     def getExecutionCode(self, required):
-        return "outTime = time - delay"
+        if self.useList:
+            return "outTimes = DoubleList.fromValues((time - delay) for delay in delays)"
```

Comment:
Might as well make the input time vectorized and both inputs virtual lists. You should implement that in Cython. See any virtual node for reference.

Reply:
Ok,I will do that in Cython.

Reply:
do you think where `c_utils` file should be placed is more better, i templary put it in `animation`, if not better, i will change.